events/teamLeave.js

```diff
@@ -0,0 +1,22 @@
+function teamLeave(context, payload) {
+    // Init variables
+    const playerId = context.id();
+    const gameState = context.getGameState();
+    const player = gameState.players[playerId]
+    const { team } = gameState.players[playerId];
+    const { name } = gameState.teams[team];
```

Comment:
Remove unused variables. Don't forget to run `npx eslint . --quiet`.

Reply:
Done and done.